<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Lucida Console,monospace</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Arial&quot;, &quot;Lucida Console&quot;, monospace;
</string>
   </property>
   <property name="text">
    <string>Batch number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>540</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	box-shadow:inset 0px 1px 0px 0px #ffffff;
	background:linear-gradient(to bottom, #ededed 5%, #dfdfdf 100%);
	background-color:#ededed;
	border-radius:6px;
	border:1px solid #dcdcdc;
	display:inline-block;
	cursor:pointer;
	color:#777777;
	font-family:Arial;
	font-size:12px;
	font-weight:bold;
	padding:6px 24px;
	text-decoration:none;
	text-shadow:0px 1px 0px #ffffff;
	
}

QPushButton:hover{
	background:linear-gradient(to bottom, #1E88E5 5%, #ffffff 100%);
	background-color:#1E88E5;
	color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>cursive</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 136, 229);
font-family: &quot;Times New Roman&quot;, Times, serif;
font-family: cursive;</string>
   </property>
   <property name="text">
    <string>StockPro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Lucida Console,monospace</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Arial&quot;, &quot;Lucida Console&quot;, monospace;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 1px solid #ced4da;
	border-radius: 8px;
	
}

QComboBox:hover{
	border: 1px solid rgb(85, 170, 255);
}

QComboBox::drop-down{
	border: 0px;
}

QComboBox::down-arrow{
	image: url(:/icon/down arrow.ico);
	width: 12px;
	height: 12px;
	margin-right: 15px;
}

/*style for QcomboBox after open select menu*/
QComboBox:on{
	border: 2px solid #c2dbfe;
}

/*style for list menu*/
QComboBox QListView{
	font-size: 12px;
	border: 1px solid rgba(0, 0, 0, 10%);
	padding: 5px;
	background-color: #fff;
	outline: 0px;
}

/*style for list of items*/
QComboBox QListView::item{
	padding-left: 10px;
	background-color: rgb(194, 219, 254);
}

QComboBox QListView::item:hover{
	background-color: rgb(194, 219, 254)
	
}</string>
   </property>
   <property name="currentText">
    <string>Appliances</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="placeholderText" stdset="0">
    <string>Type</string>
   </property>
   <item>
    <property name="text">
     <string>Appliances</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer Peripherals</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer Systems</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Electronics </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gaming</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Health &amp; Sports</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Networking</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Software &amp; Services</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Smart Home &amp; Security</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Others</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="goBack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QCommandLinkButton {

	font-family: &quot;Arial&quot;, &quot;Lucida Console&quot;, monospace;
	box-shadow:inset 0px 1px 0px 0px #ffffff;
	background:linear-gradient(to bottom, #ffffff 5%, #f6f6f6 100%);
	background-color:#ffffff;
	border:none;	

}

QCommandLinkButton:hover {
	border-bottom: 5px solid rgb(194, 219, 254);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>back.png</normaloff>back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 1px solid #ced4da;
	border-radius: 10px;

}

QLineEdit:focus{
	border: 2px solid rgb(194, 219, 254);
}

QLineEdit:hover{
	border: 1px solid rgb(85, 170, 255);
}
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Enter product name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Lucida Console,monospace</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Arial&quot;, &quot;Lucida Console&quot;, monospace;</string>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>540</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	box-shadow:inset 0px 1px 0px 0px #ffffff;
	background:linear-gradient(to bottom, #ededed 5%, #dfdfdf 100%);
	background-color:#ededed;
	border-radius:6px;
	border:1px solid #dcdcdc;
	display:inline-block;
	cursor:pointer;
	color:#777777;
	font-family:Arial;
	font-size:12px;
	font-weight:bold;
	padding:6px 24px;
	text-decoration:none;
	text-shadow:0px 1px 0px #ffffff;
	
}

QPushButton:hover{
	background:linear-gradient(to bottom, #1E88E5 5%, #ffffff 100%);
	background-color:#1E88E5;
	color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Lucida Console,monospace</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Arial&quot;, &quot;Lucida Console&quot;, monospace;</string>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Lucida Console,monospace</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Arial&quot;, &quot;Lucida Console&quot;, monospace;</string>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>661</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*QTextEdit{
	border: 2px solid #ced4da;
	border-radius: 50px;

}

QTextEdit:focus{
	border: 2px solid rgb(194, 219, 254);
}

QTextEdit:hover{
	border: 1px solid rgb(85, 170, 255);
}
</string>
   </property>
   <property name="placeholderText">
    <string>A small description of state of the product</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="statusInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 1px solid #ced4da;
	border-radius: 10px;

}

QLineEdit:focus{
	border: 2px solid rgb(194, 219, 254);
}

QLineEdit:hover{
	border: 1px solid rgb(85, 170, 255);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Is it new or used?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Lucida Console,monospace</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Arial&quot;, &quot;Lucida Console&quot;, monospace;</string>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantityInput">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 1px solid #ced4da;
	border-radius: 10px;

}

QLineEdit:focus{
	border: 2px solid rgb(194, 219, 254);
}

QLineEdit:hover{
	border: 1px solid rgb(85, 170, 255);
}
</string>
   </property>
   <property name="placeholderText">
    <string>Enter quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Lucida Console,monospace</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Arial&quot;, &quot;Lucida Console&quot;, monospace;</string>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateInput">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 1px solid #ced4da;
	border-radius: 10px;

}

QLineEdit:focus{
	border: 2px solid rgb(194, 219, 254);
}

QLineEdit:hover{
	border: 1px solid rgb(85, 170, 255);
}
</string>
   </property>
   <property name="placeholderText">
    <string>MM-DD-YYYY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Helvetica,sans-serif</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 136, 229);
font-family: Arial, Helvetica, sans-serif</string>
   </property>
   <property name="text">
    <string>Create New Product</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="batchInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 1px solid #ced4da;
	border-radius: 10px;

}

QLineEdit:focus{
	border: 2px solid rgb(194, 219, 254);
}

QLineEdit:hover{
	border: 1px solid rgb(85, 170, 255);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter batch</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>61</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icons/source.qrc"/>
 </resources>
 <connections/>
</ui>
